src/VAC/Scene.h

```diff
@@ -42,6 +42,14 @@ class InbetweenFace;
 class QDir;
 class Layer;
 
+enum struct ShapeType {
```

Comment:
I didn't even know you could write "enum struct"?   "enum class" is what's usually used in C++17's fancy enums.

Reply:
It has no relation to regular struct vs class in c++ (aka public/private by default).

Reply:
btw, best move this into VPaint namespace to avoid conflicts elsewhere. These are common names.